src/NzbDrone.Core/Indexers/NyaaTorrentRssParser.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Linq;
+using System.Xml.Linq;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.Indexers.Exceptions;
+
+namespace NzbDrone.Core.Indexers
+{
+    public class NyaaTorrentRssParser : TorrentRssParser
+    {
+        public NyaaTorrentRssParser()
+        {
+        }
+
+        protected override long GetSize(XElement item)
```

Comment:
`TorrentRssParser` can already parse this by setting `SizeElementName` and having it strip the name spaces there.